lisa/features/sriov.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any
+
+from lisa.feature import Feature
+from lisa.platform_ import Platform
+from lisa.util.logger import get_logger
+
+FEATURE_NAME_SRIOV = "Sriov"
+
+
+class Sriov(Feature):
+    def __init__(self, node: Any, platform: Any) -> None:
+        super().__init__(node, platform)
+        self._log = get_logger("feature", self.name(), self._node.log)
+
+    @classmethod
+    def name(cls) -> str:
+        return FEATURE_NAME_SRIOV
+
+    def _switch_sriov(self, platform: Platform, enable: bool) -> None:
+        raise NotImplementedError()
+
+    def enable_sriov(self) -> None:
```

Comment:
Because the methods are in sriov already, their names doesn't need to have sriov.